### PR TITLE
Custom threshold for PROBE_POINTS_INCREASING check on QGL and Z_TILT

### PR DESCRIPTION
**Pull request by Zeanon**
As mentioned by some people, a way to adjust the threshold would be preferred over just ignoring the check, so here we go

### CODE REVIEW

**Comment by rogerlz** [comment]
That's really nice!

Can you please fix the conflict and squash the commits?
I will add some comments to the wording

File: README.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ If I want my printer to light itself on fire, I should be able to make my printe
 
 - [canbus: custom CAN bus uuid hash for deterministic uuids](https://github.com/DangerKlippers/danger-klipper/pull/156)
 
+- [z_tilt and qgl: ignore probe_points_increasing check](https://github.com/DangerKlippers/danger-klipper/pull/183)
```
Comment (rogerlz) [inline README.md:89]: I'd say `threshold for probe_points_increasing check`

File: docs/Config_Reference.md [hunk 2]
```diff
@@ -1308,6 +1308,10 @@ extended [G-Code command](G-Codes.md#z_tilt) becomes available.
 #   more points than steppers then you will likely have a fixed
 #   minimum value for the range of probed points which you can learn
 #   by observing command output.
+#increasing_threshold: 0.0000001
+#   Sets the threshold that probe points can increase before the z_tilt
+#   will be aborted.
+#   If you want to ignore the check just set it really high (like 9999)
```
Comment (rogerlz) [inline docs/Config_Reference.md:1314]:
```
#   Sets the threshold that probe points can increase before z_tilt aborts.
#   To disable the validation, set this parameter to a high value.
#   The default is 0.0000001.
```

Comment (Zeanon) [reply docs/Config_Reference.md:1314]: Fair enough, I just didn't specify the default in the text as it's already specifies on the parameter(as Kevin normally does it)

**Comment by Zeanon** [comment]
Where the hell did this conflict come from, I actually just merged my master on my fork with DK master and then based the branch off of that, well maybe gut f'ed up.
Well I'll take a look at it once I'm home

File: README.md (hunk 1 above)
Comment (Zeanon) [reply README.md:89]: Oh yeah forgot to edit that, mb

File: docs/Config_Reference.md (hunk 2 above)
Comment (rogerlz) [reply docs/Config_Reference.md:1314]: yeah that works

**Review by rogerlz — APPROVED** [review]

**Pull request merged**